prow/config.yaml

```diff
@@ -341,6 +341,30 @@ tide:
     - do-not-merge/hold
     - do-not-merge/work-in-progress
     reviewApprovedRequired: true
+  - repos:
```

Comment:
why not just the whole istio org?

Reply:
No particular reason AFAIK. I copied down the query repos from above.